tools/automation.py

```diff
@@ -0,0 +1,316 @@
+"""
+    Create a script that will convert mentor's data from mentors.xlsx to mentors.yml file
+
+    Run script:
+        \tools>run_mentors_automation.bat
```

Comment:
```suggestion
        \tools>run_mentors_automation.sh
```

Reply:
Removed.

Reply:
Could you push the changes? because I don't see it here.